app/controllers/songs_controller.rb

```diff
@@ -17,7 +17,7 @@ def show
   end
 
   def create
-    @song = Song.create(song_params)
+    @song = Song.create(song_params.merge(user_id: current_user.id))
```

Comment:
No. It's difficult. You can use association 😉 
```
@song = current_user.songs.create(song_params)
```
The way above is a good way. BUT! I advise a different way. Use callback in Song model:
```
before_create: # and add user_id
```